src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch("https://my-nc-news-yyln.onrender.com/api/articles")
+      .then((response) => {
+        return response.json();
+      })
+      .then((articleData) => {
+        console.log(typeof articleData);
+        setArticles(articleData.articles);
+      });
+  }, []);
```

Comment:
check out the docs for making API calls using [axios](https://l2c.northcoders.com/courses/sd-notes/front-end#sectionId=axios,step=intro) , in the long run it's much easier to work with Axios than fetch 👍